seismic_zfp/utils.py

```diff
@@ -25,6 +25,14 @@ def __init__(self, data_array, ilines, xlines, samples):
         self.samples = samples
 
 
+class Axes:
+    """Utility class to hold axes"""
+    def __init__(self, ilines, xlines, samples):
+        self.ilines = ilines
+        self.xlines = xlines
+        self.samples = samples
```

Comment:
sounds good, I will investigate

Reply:
I have been thinking more about this and after looking at the rest of the code I come to the conclusion to not do it. None of the code uses immutability while there are lots of utility classes that could use it. the main reason is to keep the overall code style consistent. it would read very strange if one specific util class uses it while all the others don't

Reply:
Yes I agree